relay_rpc/src/auth/cacao/signature/eip1271/blockchain_api.rs

```diff
@@ -0,0 +1,59 @@
+use {super::get_rpc_url::GetRpcUrl, crate::domain::ProjectId, url::Url};
+
+// https://github.com/WalletConnect/blockchain-api/blob/master/SUPPORTED_CHAINS.md
+const SUPPORTED_CHAINS: [&str; 26] = [
```

Comment:
I considered checking the response of Blockchain API to determine support or not, but this would make the CACAO verification function more coupled to Blockchain API so decided to hardcode the supported list here for now.

In the future we should either pull the list dynamically, or refactor the CACAO to use a more generic error handling and be able to interpret the response code. I slightly prefer the check-before-request approach to avoid redundant requests.